spec/support/shoulda_matchers.rb

```diff
@@ -0,0 +1,8 @@
+require 'shoulda/matchers'
+
+Shoulda::Matchers.configure do |config|
+  config.integrate do |with|
+    with.test_framework :rspec
+    with.library :rails
+  end
+end
```

Comment:
This file is not really needed, shoulda matchers infers it. But I don't mind making it explicit.

Reply:
Never mind me, this is needed I just added it directly on the file